docs/Mods.md

```diff
@@ -31,6 +31,7 @@ declare type Mods = {|
     displayNameLabel?: string,
     descriptionLabel?: string,
     inputTypeLabel?: string,
+    placeHolderLabel?: string,
```

Comment:
Apologies to request such a menial change, but `placeHolderLabel` is incorrectly cased. It should be `placeholderLabel` (in this and a handful of other files).

Other than that, looks good to me. I can merge once that's ready.